### PR TITLE
merge `fragment-elements` and `fragment-gateway` into a single package (`web-fragments`)

### PR DESCRIPTION
**Pull request by dario-piotrowicz**

### CODE REVIEW

File: e2e/pierced-react/README.md [hunk 1]
```diff
@@ -3,7 +3,6 @@
 To run the app (that shows the server side piercing) you need to run (in order):
  - `pnpm i` at the root of the monorepo
  - `pnpm build` in /reframed
```
Comment (IgorMinar) [inline e2e/pierced-react/README.md:5]: can you please update these steps in the following style:
```suggestion
 - `pnpm --filter reframed build`
 - `pnpm --filter ./packages/web-fragments build`
 - `pnpm --filter ./e2e/pierced-react-remix-fragment build && pnpm --filter ./e2e/pierced-react-remix-fragment start`
 - new terminal: `pnpm --filter ./e2e/pierced-react dev`
```

**Review by IgorMinar — APPROVED** [review]
I tested this manually and still works!

File: e2e/pierced-react/README.md (hunk 1 above)
Comment (dario-piotrowicz) [reply e2e/pierced-react/README.md:5]: what about:
```
 - `pnpm i`
 - `pnpm --filter reframed --filter ./packages/web-fragments --filter ./e2e/pierced-react-remix-fragment build`
 - `pnpm --filter ./e2e/pierced-react-remix-fragment start`
 - in a new terminal: `pnpm --filter ./e2e/pierced-react dev`
```
?

🙂

Comment (IgorMinar) [reply e2e/pierced-react/README.md:5]: that works! thanks

Comment (IgorMinar) [reply e2e/pierced-react/README.md:5]: (works much better)

**Pull request merged**